### PR TITLE
Add database config storage for all 'da' configs

### PR DESCRIPTION
**Pull request by pwirth**
issue: https://github.com/inblockio/DataAccounting/issues/176

### CODE REVIEW

**Review by FantasticoFox — COMMENTED** [review]
Looked over it and really appreciate the clean code with inclusion of types. Wait for proper feedback from @rht for merge

**Comment by rht** [comment]
Merged, thank you!

File: includes/Config/DataAccountingConfig.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+use Config;
+use Exception;
+use MultiConfig;
+use MutableConfig;
+
+class DataAccountingConfig extends MultiConfig implements MutableConfig {
+	/**
+	 * @var Handler 
+	 */
+	protected $handler = null;
```
Comment (rht) [inline includes/Config/DataAccountingConfig.php:14]: @pwirth while fixing psalm type annotation stuff, it complains here that
```
ERROR: PossiblyNullPropertyAssignmentValue - includes/Config/DataAccountingConfig.php:14:23 - $this->handler with non-nullable declared type 'DataAccounting\Config\Handler' cannot be assigned nullable type 'null' (see https://psalm.dev/148)
```

Can I safely delete the `= null` here?

Comment (rht) [reply includes/Config/DataAccountingConfig.php:14]: OK, I found that it is not possible. Should the type be `Handler?` then?

**Pull request merged**